src/Microsoft.SqlTools.ServiceLayer/Workspace/WorkspaceService.cs

```diff
@@ -263,7 +265,7 @@ public void RegisterTextDocOpenCallback(TextDocOpenCallback task)
                 }
```

Comment:
Do we need to do this anymore if we're passing the URI in directly? When would openedFile be null anyways?